src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -239,6 +251,16 @@ private TraceWriter CreateDefaultTraceWriter(WebHostSettings settings)
             return new CompositeTraceWriter(new[] { systemTraceWriter, fileTraceWriter });
         }
 
+        private ILoggerFactory CreateDefaultLoggerFactory(WebHostSettings settings)
+        {
+            var config = GetScriptHostConfiguration(settings);
```

Comment:
Creating a factory completely detached from the active ScriptHostConfiguration because I don't want to impact the initialization of the actual logger factory, which may be happening in parallel.